app/views/install_stable.scala.txt

```diff
@@ -9,10 +9,11 @@ SDKMAN_VERSION="@cliVersion"
 SDKMAN_PLATFORM=$(uname)
 
 if [ -z "$SDKMAN_DIR" ]; then
-    SDKMAN_DIR="$HOME/.sdkman"
+    SDKMAN_DIR=$(readlink -f "$HOME/.sdkman")
```

Comment:
If we are going to allow these changes, they should only be done in the beta hook until we can verify that they work on all platforms in the beta channel.

Reply:
I'm not sure if `readlink -f` works on Mac without installing `coreutils`.